rust-symcrypt/BUILD.md

```diff
@@ -26,18 +28,18 @@ You can configure your system one of 3 ways.
     ```
     **Note:** This change will only persist within the current process, and you must re-set the PATH environment variable after closing the PowerShell window.
 
-2. Manually copy the `symcrypttestmodule.lib` to `C:\Windows\System32`
-    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypttestmodule.lib`
+2. Manually copy the `symcrypt.lib` to `C:\Windows\System32`
+    Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`
 
-3. Permanently add the lib path into your system PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypttestmodule.lib`
+3. Permanently add the lib path into your system PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`
 
 **Option 1 or Option 2 is what is recommended for ease of use.**
 
-The `symcrypttestmodule.dll` can be found in the the following path after SymCrypt has been successfully downloaded and built. 
+The `symcrypt.dll` can be found in the the following path after SymCrypt has been successfully downloaded and built. 
 
 `C:\Your-Path-To-SymCrypt\SymCrypt\bin\exe`

```

Comment:
This path is also wrong, it should be in the `dll` path relative to the archive root or the build output root.